engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/error/ErrorEventTest.java

```diff
@@ -143,6 +144,135 @@ public void shouldCatchErrorEventsByErrorCode() {
         .containsOnly("error-2");
   }
 
+  @Test
```

Comment:
:+1: Great test coverage!

We should also add a test case to assert that a code-specific Error catch event is prefered, when both a code-specific and a catch-all Error catch events are present.